package.json

```diff
@@ -23,6 +23,7 @@
     "autoprefixer": "^9.8.6",
     "babel-loader": "^8.2.2",
     "browser-sync": "^2.26.14",
+    "google-protobuf": "^3.15.8",
```

Comment:
Please check for licensing, MIT is good

Reply:
BSD-3-Clause

Source:
https://www.npmjs.com/package/google-protobuf